app.js

```diff
@@ -4,19 +4,21 @@ const express = require("express");
 const cors = require("cors");
 const morgan = require("morgan");
 const dotenv = require("dotenv");
+const routes = require("./routes");
 
 const app = express();
+const PORT = process.env.PORT;
```

Comment:
const 키워드로 생성한 변수의 경우 블록스코프의 범위를 갖습니다.
그렇기 때문에 보통 해당 변수가 사용되는 함수 바로 위에서 선언하여 값을 할당한 뒤 사용합니다.

가장 좋은 방법은 해당 변수가 사용되는 블록 내부에서 선언하는 것입니다.
현재 `PORT`라는 변수는 start 함수 내부에서 사용되기 때문에 아래와 같이 수정해주면 좋습니다:
```suggestion
const start = () => {
  const PORT = process.env.PORT;
  app.listen(PORT, () => {
    console.log(' ... ');
  });
}
```